internal/stacks/stackplan/from_proto.go

```diff
@@ -204,6 +206,79 @@ func LoadFromProto(msgs []*anypb.Any) (*Plan, error) {
 				c.ResourceInstancePriorState.Put(fullAddr, nil)
 			}
 
+		case *tfstackdata1.PlanDeferredResourceInstanceChange:
```

Comment:
There's a lot of overlap between this case and the planned change, which makes sense. Do you think there might be some way to unify them, maybe with helper functions? I'm a bit concerned about logic drift over time if the code is duplicated.

Reply:
Done!